src/patternfly/components/Nav/nav.scss

```diff
@@ -203,6 +207,34 @@
   }
 
   position: relative;
+  display: flex;
+  flex-wrap: wrap;
+  overflow: hidden;
+
+  &.pf-m-scrollable {
```

Comment:
I wonder if you even need this modifier, because if the `nav__scroll-button`s only show up when the nav is scrollable, then couldn't the styles be applied to them regardless. I would see `pf-m-scrollable` being necessary if it styled the actual nav.